### PR TITLE
PYTHON-968: Support IS NOT NULL operator in cqlengine 

### PR DESCRIPTION
**Pull request by bjmb**

### CODE REVIEW

File: tests/integration/cqlengine/operators/test_where_operators.py [hunk 1]
```diff
@@ -48,3 +59,56 @@ def test_operator_rendering(self):
         self.assertEqual("<=", six.text_type(LessThanOrEqualOperator()))
         self.assertEqual("CONTAINS", six.text_type(ContainsOperator()))
         self.assertEqual("LIKE", six.text_type(LikeOperator()))
+
+
+class TestIsNotNull(BaseCassEngTestCase):
+    def test_is_not_null_to_cql(self):
```
Comment (mambocab) [inline tests/integration/cqlengine/operators/test_where_operators.py]: nit: Could this be a unit test?

Comment (bjmb) [reply tests/integration/cqlengine/operators/test_where_operators.py]: It definitely can, but we don't have unit tests for cqlengine, so I figured at some point we didn't want to create a new folder just for a few tests. They don't start C* so they don't add more time to the run for being there.
If we moved them to unit I'd do so in a future PR

**Comment by bjmb** [comment]
They track count of how many parameters have been processed because the way cqlengine builds the queries is:
`SELECT clolumn1, column2, ... from keyspace.table WHERE A = %(0)s AND  B = %(1)s`, `parameters = {0: "parameter for A", 1: "parameter for B"}`. 
Using 0, 1, 2, ... to track the parameters.

So each clause need to know how many parameters the clauses from before have taken so they can put the "0" or the "1" or the number that it corresponds to them. Since `IS NOT NULL` uses zero `get_context_size` returns 0 and in `update_context` the context is not updated since it doesn't affect the global count nor uses parameters.

File: tests/integration/cqlengine/operators/test_where_operators.py (hunk 1 above)
Comment (mambocab) [reply tests/integration/cqlengine/operators/test_where_operators.py]: :+1: awesome. filed a low-priority ticket: https://datastax-oss.atlassian.net/browse/PYTHON-975

**Comment by mambocab** [comment]
Ohhh, cool. Thanks for the explanation.

**Comment by mambocab** [comment]
Looks great, +1 for merging.

**Comment by bjmb** [comment]
Awesome, thank you for the review!

File: docs/cqlengine/queryset.rst [hunk 2]
```diff
@@ -199,6 +199,14 @@ In addition to simple equal to queries, cqlengine also supports querying with ot
 
         q = Automobile.objects.filter(model__like='%Civic%').allow_filtering()
 
+:attr:`IS NOT NULL (IsNotNull(column_name)) <query.QueryOperator.IsNotNullOperator>`
```
Comment (aboudreault) [inline docs/cqlengine/queryset.rst]: This should probably point to the Clause and not the Operator. In this case, I think the Operator can't be used directly.

Comment (bjmb) [reply docs/cqlengine/queryset.rst]: That's correct, the clause has to be used for this case. I'll change it

**Pull request merged**